client/src/components/planner/DegreePlanner.js

```diff
@@ -1,10 +1,65 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
+import Select from "react-select";
+import {makeStyles} from "@material-ui/core";
+
+const useStyles = makeStyles({
+    searchBar: {
+        width: "80%",
+        margin: "auto",
+        paddingTop: "2%"
+    }
+});
 
 export default function DegreePlanner() {
+    const classes = useStyles();
+
+    const [course, setCourse] = useState({});
+    const [prereqs, setPrereqs] = useState([]);
```

Comment:
For the sake of processing and component complexity, lets remove the prereq and postreq logic so for now it just calls upon course endpoint and logs selected courses.

Reply:
prereq and postreq logic removed